fastapi_app/python/opt_models/base_optimizer.py

```diff
@@ -42,7 +42,9 @@ def __init__(
         self.user_id = user_id
         self.project_id = project_id
         n_days = min(self.project_setup["n_days"], int(os.environ.get('MAX_DAYS', 365)))
-        self.start_datetime = pd.to_datetime(self.project_setup["start_date"]).to_pydatetime()
+        # self.start_datetime = pd.to_datetime(self.project_setup["start_date"]).to_pydatetime()
+        # start_datetime hardcoded as only 2022 pv and demand data is available
+        self.start_datetime = pd.to_datetime('2022').to_pydatetime()
```

Comment:
@ha-cra 
What will then happen if the data gets updated? What would be the best way of remembering such an hardcoded fix? 

@GregoryIreland  - do you have suggestions? This conversation has a link that can be put in some report file or a github issue for reference